app/pages/sections/order-items/catalogue-solutions/select/routes.js

```diff
@@ -242,7 +255,6 @@ export const catalogueSolutionsSelectRoutes = (authProvider, addContext, session
       req,
       key: sessionKeys.selectedRecipients,
     });
-
```

Comment:
As above, I think it is better to keep the blank line here.

Reply:
done